src/backend/opencl/platform.cpp

```diff
@@ -529,7 +521,7 @@ void removeDeviceContext(cl_device_id dev, cl_context ctx) {
     {
         common::lock_guard_t lock(devMngr.deviceMutex);
 
-        const int dCount = devMngr.mDevices.size();
+        const size_t dCount = devMngr.mDevices.size();
         for (int i = 0; i < dCount; ++i) {
```

Comment:
```suggestion
        for (size_t i = 0; i < dCount; ++i) {
```

Reply:
Here I get now a warning at the deleteIdx assignment, which has to remain int.

As above, I will go back to the original with an extra static_cast<int>.